src/haddock/modules/__init__.py

```diff
@@ -43,6 +43,7 @@
     "scoring",
     "analysis",
     "extras",
+    "design"
```

Comment:
This list is ordered by workflow progression. You may wish to add `design` in a different position.